src/main/java/io/anserini/collection/TwentyNewsgroupsCollection.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Anserini: A Lucene toolkit for replicable information retrieval research
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.collection;
+
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Path;
+import java.util.NoSuchElementException;
+
+/**
+ * An instance of the <a href="http://people.csail.mit.edu/jrennie/20Newsgroups">20 Newsgroup by date</a>.
```

Comment:
I don't think you resolved this comment? What does "by date" mean? Same below also.

Reply:
by date is for the data set version. There're in total three different 20Newsgroup version, and by date is the one we used. Maybe I shouldn't include that dataset version there, I added it in the comment below that line now.